test/compflags/bradc/printstuff/versionhelp.sh

```diff
@@ -5,4 +5,4 @@ compiler=$3
 echo -n `basename $compiler`
 cat $CWD/version.goodstart
 diff $CWD/../../../../compiler/main/BUILD_VERSION $CWD/zero.txt > /dev/null 2>&1 && echo "" || \
-    { echo -n "." && cat $CWD/../../../../compiler/main/BUILD_VERSION | tr -d \" ; }
\ No newline at end of file
+    { echo -n " pre-release (" && cat $CWD/../../../../compiler/main/BUILD_VERSION | tr -d \"\\n && echo ")" ; }
```

Comment:
I hadn't noticed that the final linefeed was dropped here on one of the previous few commits, and it doesn't seem to have hurt anything (which surprises me slightly), but I think we should re-instate it anyway.

That was the only thing I noticed that seemed amiss.  I'm running a full test run on the branch to check for other surprises.

Reply:
Testing came back clean, so I think that's the only thing.  Thanks!